models/release.go

```diff
@@ -142,6 +142,12 @@ func createTag(gitRepo *git.Repository, rel *Release) error {
 		if err != nil {
 			return fmt.Errorf("CommitsCount: %v", err)
 		}
+
+		u, err := GetUserByEmail(commit.Author.Email)
+		if err == nil {
+			rel.PublisherID = u.ID
```

Comment:
Doesn't this needs also migration for old tags

Reply:
Haha, true.

Will fix

Reply:
See https://github.com/go-gitea/gitea/pull/6025/commits/09fef968ee0e0801e7a518fe0241db2af6270a65